website/static/js/accountSettings.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+var $ = require('jquery');
+var $osf = require('js/osfHelpers');
+var ko = require('knockout');
+var oop = require('js/oop');
+var Raven = require('raven-js');
+
+require('knockout.punches');
+ko.punches.enableAll();
+
+
+var UserEmail = oop.defclass({
+    constructor: function() {
```

Comment:
Might make sense to pass in params into the constructor so that you can construct a `UserEmail` without having to know which fields are observables and set those observables manually (in `UserProfile#unserialize`).